packages/tizen_bundle/lib/tizen_bundle.dart

```diff
@@ -0,0 +1,318 @@
+
+    return keys;
+  }
+
+  /// Deletes the item from the bundle object with the specific key.
+  void delete(String key) {
+    int ret = tizen.bundle_del(_handle, key.toNativeInt8());
+    if (ret != bundle_error_e.BUNDLE_ERROR_NONE) {
+      Log.error(_logTag, "Failed to delete key/value. key: $key, error: $ret");
+      _BundleErrorFactory().throwException(ret);
+    }
+  }
+
+  /// Checks whether the bundle object is empty or not.
+  bool isEmpty() {
+    return tizen.bundle_get_count(_handle) == 0;
+  }
+
+  /// Adds a bytes type key-value pair into the bundle object.
+  void addBytes(String key, List<int> bytes) {
+    using((Arena arena) {
+      final byteArray = arena<Uint8>(bytes.length);
+      for (int index = 0; index < bytes.length; ++index) {
+        byteArray[index] = bytes[index] & 0xff;
+      }
+
+      int ret = tizen.bundle_add_byte(
+          _handle, key.toNativeInt8(), byteArray.cast<Void>(), bytes.length);
+      if (ret != bundle_error_e.BUNDLE_ERROR_NONE) {
+        Log.error(_logTag, "Failed to add bytes. key: $key, error: $ret");
+        _BundleErrorFactory().throwException(ret);
+      }
+    });
+  }
+
+  /// Gets bytes from the bundle object with the given specific key.
+  List<int> getBytes(String key) {
+    List<int> values = using((Arena arena) {
+      final bytes = arena<Pointer<Uint8>>();
+      final bytesSize = arena<Int32>();
+      int ret = tizen.bundle_get_byte(
+          _handle, key.toNativeInt8(), bytes.cast<Pointer<Void>>(), bytesSize);
+      if (ret != bundle_error_e.BUNDLE_ERROR_NONE) {
+        Log.error(_logTag, "Failed to get bytes. key: $key, error: $ret");
+        _BundleErrorFactory().throwException(ret);
+      }
+
+      List<int> byteList = [];
+      for (int index = 0; index < bytesSize.value; ++index) {
+        byteList.add(bytes.value[index]);
+      }
+
+      return byteList;
+    });
+
+    return values;
+  }
+
+  /// Converts this object to BundleRaw type.
+  BundleRaw toRaw() {
+    BundleRaw raw = using((Arena arena) {
+      final rawPointer = arena<Pointer<Int8>>();
+      final lengthPointer = arena<Int32>();
+      int ret = tizen.bundle_encode(
+          _handle, rawPointer.cast<Pointer<Uint8>>(), lengthPointer);
+      if (ret != bundle_error_e.BUNDLE_ERROR_NONE) {
+        Log.error(_logTag, "Failed to encode bundle to raw data. error: $ret");
+        _BundleErrorFactory().throwException(ret);
+      }
+
+      return BundleRaw(rawPointer.value.toDartString(), lengthPointer.value);
+    });
+
+    return raw;
+  }
+
+  /// Gets the number of items in the bundle object.
+  int getCount() {
+    return tizen.bundle_get_count(_handle);
+  }
+
+  /// Gets a type of the item of the key.
+  int getType(String key) {
+    return tizen.bundle_get_type(_handle, key.toNativeInt8());
+  }
+
+  /// Checks whether the key exists or not.
+  bool contains(String key) {
+    return (tizen.bundle_get_type(_handle, key.toNativeInt8()) !=
+        BundleType.none) as bool;
```

Comment:
Remove unnecessary cast
```suggestion
    return tizen.bundle_get_type(_handle, key.toNativeInt8()) != BundleType.none;
```

Reply:
Done